tests/e2e/testnet-scripts/start-chain.sh

```diff
@@ -375,16 +375,18 @@ fi
 
 
 # poll for chain start
-if [[ "$USE_COMETMOCK" == "true" ]]; then
-    set +e
-    until $BIN query block --type=height 0 --node "tcp://$CHAIN_IP_PREFIX.$QUERY_IP_SUFFIX:26658"; do sleep 0.3 ; done
-    set -e
-else
-    set +e
-       until $BIN query block --type=height 0 --node "tcp://$CHAIN_IP_PREFIX.$QUERY_IP_SUFFIX:26658" | grep -q -v '{"block_id":{"hash":"","parts":{"total":0,"hash":
-    ""}},"block":null}'; do sleep 0.3 ; done
-    set -e
-fi
+# if [[ "$USE_COMETMOCK" == "true" ]]; then
+#     set +e
+#     until $BIN query block --type=height 0 --node "tcp://$CHAIN_IP_PREFIX.$QUERY_IP_SUFFIX:26658"; do sleep 0.3 ; done
+#     set -e
+# else
+#     set +e
+#        until $BIN query block --type=height 0 --node "tcp://$CHAIN_IP_PREFIX.$QUERY_IP_SUFFIX:26658" | grep -q -v '{"block_id":{"hash":"","parts":{"total":0,"hash":
```

Comment:
The `height must be greater than 0` error seems to stem from this [bump](https://github.com/cosmos/interchain-security/pull/2200). I think we could have it `--type=height 1` and keep it as it was.

Reply:
This fixes the issue. Thanks!